### DISCUSSION TITLE
How to disable showing the first line of a code block?

### DISCUSSION THREAD

**Discussion by Xiaobo2020** [LazyVim/LazyVim#1938] [Q&A] [answered]
I don’t want to display the first line of the code block where the cursor is on the first line. How can I disable this feature? 

As shown in the screenshot, there is clearly folded content between the first and second lines. I don't want this folding feature to take effect. 

I'm sorry that I can't find a more professional word to describe this feature.

<img width="1815" alt="image" src="https://github.com/LazyVim/LazyVim/assets/16217316/52e059db-e80c-43a6-a6df-a24bfe89617e">

**Comment by dpetka2001** [comment, marked as answer]
It's `nvim-treesitter/nvim-treesitter-context`. You can either disable the plugin or there's a keymap `<leader>ut` to temporarily toggle the feature on/off.

**Reply by Xiaobo2020** [reply]
Thanks!

**Comment by AndreasNasman** [comment]
I wanted to keep the plugin active but disable it on startup, so I ended up doing

```lua
"nvim-treesitter/nvim-treesitter-context", opts = { enable = false }
```
